source/rules/roles.txt

```diff
@@ -266,231 +323,153 @@ the order in which they are evaluated and applied. Each role's apply
-   <data-sources>`. Roles and permissions :ref:`do not apply
-   <data-federation-caveats>` to {+adf-datasource+}s.
-
 .. tabs-realm-admin-interfaces::
-   
+
    .. tab::
       :tabid: ui
-      
-      .. procedure::
-
-         .. step:: Navigate to the Rules Window
-
-            Under :guilabel:`Data Access` in the left navigation menu, click :guilabel:`Rules`. 
-
-         .. step:: Configure Roles for a Collection
-
-            A. For the linked cluster that you want to define access permissions for,
-               click the ellipsis :guilabel:`(...)` and select :guilabel:`Create a collection`.
-            
-            #. In the modal, enter the :guilabel:`Database Name` and 
-               :guilabel:`Collection Name` of the MongoDB collection that you want to use.
-
-            .. note:: 
-
-               Alternatively, you can select :guilabel:`Default roles and filters` under
-               the linked cluster to configure :ref:`default rules <default-rules>` 
-               for the entire cluster.
 
-         .. step:: Create a New Role
-
-            To create a new role, click :guilabel:`Add role`. You can
-            either start from scratch or use a preset role as a starting
-            point.
-
-         .. step:: Configure the Role
-
-            A. Give the role a name. The name can be anything you want but
-               must be unique within a given collection. Consider using names
-               that describe the user (e.g. ``Admin``) and/or their
-               relationship to the data (e.g. ``Owner``).
-
-            #. Define the Role's :guilabel:`Apply When` Condition. 
-               A role's :ref:`Apply when <apply-when-expressions>` condition 
-               determines when a given user has the role for a given document.
-
-            #. Define the role's :guilabel:`Document Permissions`. For more 
-               information, see :ref:`document-level-permissions`. 
-
-            #. Define the role's :guilabel:`Field Permissions`. For more 
-               information, see :ref:`field-level-permissions`. 
-
-         .. step:: Save the Role
-
-            Save the role. Once you have finished defining roles for the collection, 
-            App Services will immediately begin using the roles you
-            defined for all incoming queries on the collection.
-
-            .. note::
+      1. Click :guilabel:`Rules` in the left navigation menu and then select a
+         collection from the data source's collection list. You can also select
+         :guilabel:`Default roles and filters` to configure default rules for
+         the entire cluster.
+      
+         If no roles are defined yet, you'll be prompted to create a new
+         one. Otherwise you'll see an ordered list of existing roles.
+         
+      #. To create a new role, click :guilabel:`Add role`. You can
+         either start from scratch or use a preset role as a starting
+         point.
```

Comment:
Let's keep these steps parallel by removing the infinitive. Creating a new role is the implied goal of this procedure. If it's not clear enough, we should add a procedure title above all the steps.

We also probably want to avoid the "start from scratch" idiom, as it may be harder for non-native English speakers to understand.

```suggestion
      #. Click :guilabel:`Add role` to define a new role or or use a 
         preset role as a starting point.
```